Gopkg.lock

```diff
@@ -1,15 +1,30 @@
 # This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
 
 
+[[projects]]
+  branch = "master"
+  name = "github.com/Azure-Samples/azure-sdk-for-go-samples"
+  packages = [
+    "authorization",
+    "helpers",
+    "iam",
+    "network",
+    "resources"
+  ]
+  revision = "819a10796e6495b7292add979375cff8c247f927"
+
 [[projects]]
   name = "github.com/Azure/azure-pipeline-go"
   packages = ["pipeline"]
-  revision = "f4da77e3846319876aebb41f2e6dccc6b1d7f1e0"
-  version = "1.1.3"
+  revision = "9804b770d6810999e2bcb7480fb509504e72b2ba"
+  version = "0.1.4"
 
 [[projects]]
   name = "github.com/Azure/azure-sdk-for-go"
   packages = [
+    "arm/authorization",
```

Comment:
we should avoid the "arm/" packages since they're deprecated

Reply:
This is used by the `authorization/auth.go` sample and requires fixing there.